test/bugle_forms/factories.clj

```diff
@@ -0,0 +1,38 @@
+(ns bugle-forms.factories
+  (:require
+   [bugle-forms.specs :as specs]
+   [clojure.spec.alpha :as s]
+   [clojure.spec.gen.alpha :as gen]))
+
+(defn user
+  "Randomly generate a user.
+  Optionally takes a map of overrides."
+  ([] (user {}))
+  ([overrides]
+   (gen/generate
```

Comment:
[Note] It's better to return a generator from the factory instead of the realized value using `gen/generate`. This will allow you to compose these factories easily and make bigger factories from it.